releases/latest/kernel/helpers/build/configuration_snippets/truststore.xml

```diff
@@ -0,0 +1,4 @@
+<server description="Default Server">
+    <ssl id="defaultSSLConfig" keyStoreRef="defaultKeyStore" trustStoreRef="defaultTrustStore"/>
```

Comment:
should we also add `trustDefaultCerts=“true”` here?

Reply:
Yes, I can add that. Also it also I think is applicable when there is no custom truststore.
So I think we will need 2 versions  <ssl id="defaultSSLConfig"> depending on if there is a truststore.